src/functions-and-arrays.js

```diff
@@ -52,15 +112,37 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(uniqueArray) {
+  let simpleArray = [];
+  if (uniqueArray.length === 0){
+    return null;
+  } else {
+    uniqueArray.forEach ((word)=>{
+      if (simpleArray.includes(word) === false){
+      simpleArray.push(word);
+      }
+  });
+  return simpleArray;
+  }
+}
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+  function doesWordExist(wordsFind, onlyone) {
+  if (wordsFind === 0){
```

Comment:
In this iteration you're failing only one test, and this is happening because here you're checking if an array is equal to a number `wordsFind === 0` but what you really want to check is if the length of the array is equal to zero, so what you need to do is `wordsFind,length === 0`